manage.py

```diff
@@ -160,6 +181,43 @@ def pull_prod_data():
         print("Finished copying upload files")
 
 
+def _create_default_users_with_password(password_for_default_users):
+    environment = os.environ.get("ENVIRONMENT", "PRODUCTION")
+    if environment == "PRODUCTION":
```

Comment:
maybe do a `.upper()` on this before the comparison?

Reply:
Yep, done.